src/server/api/trpc.ts

```diff
@@ -38,11 +38,12 @@ interface CreateContextOptions {
  *
  * @see https://create.t3.gg/en/usage/trpc#-serverapitrpcts
  */
+
 const createInnerTRPCContext = (opts: CreateContextOptions) => {
   return {
     session: opts.session,
-    prismaPostgres,
-    prismaMongo,
+    prismaPostgres: prismaPostgres,
```

Comment:
I don't think this change is needed?

Reply:
Yea this is an oops, I'll change it back